waggle-dance-core/pom.xml

```diff
@@ -187,6 +179,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    
+    <!-- Hive common -->
```

Comment:
Well, it is actually Hive common as it contains a bunch of generic code to interact with the Metastore. It does also contains the code that allows one to communicate with a metastore via an SSH tunnel but the library is broader than just that.

Reply:
We already have a <--hive--> section where we have Hive related Libraries added. IMHO, having it separately tagged as <--Hive-common--> is a bit confusing and doesn't give the reader context as to what purpose its being added for in the project. Is there any other use for this library in this project other than SSH tunnel ?

Reply:
Yes, there is another use for this library. https://github.com/HotelsDotCom/circus-train/pull/76 is an example of what it can help with, and something similar to this will happen for WaggleDance too.

Reply:
Alright. I will resolve this.

Reply:
I see what you're saying about "Hive common" being confusing. I think we should just remove the comment and assume that the name of the artifact is sufficient documentation ;) The "Hive" section was probably intended to group all the Apache Hive artifacts, not our own Hive-related ones.